R/visualization/plotBestMethodsATAC.R

```diff
@@ -67,7 +60,7 @@ plotBestMethodsATAC<- function(csv_atac_small_path, csv_atac_large_path){
     methods_name <- capitalize(methods_name)
     methods_name <- plyr::mapvalues(methods_name, 
                                     from = c("Seurat", "Seuratrpca", "Mnn", "Bbknn", "Trvae", "Scvi", "Liger", "Combat", "Saucie", "Fastmnn", "Desc", "Scanvi", "Scgen"), 
-                                    to = c("Seurat v3 CCA", "Seurat v3 RPCA", "MNN", "BBKNN", "trVAE", "scVI", "LIGER", "ComBat", "SAUCIE", "fastMNN", "DESC", "scANVI", "scGen"))
+                                    to = c("Seurat v3 CCA", "Seurat v3 RPCA", "MNN", "BBKNN", "trVAE", "scVI", "LIGER", "ComBat", "SAUCIE", "fastMNN", "DESC", "scANVI*", "scGen*"))
```

Comment:
There are a few mappings like this that are reused in the various files. I'm wondering whether they could go in one place so that we can make sure they are consistent? That's what I have tried to do here for the website https://github.com/theislab/scIB-reproducibility/blob/main/website/R/load.R#L138-L181.